### ISSUE TITLE
[TUI] Kitty graphics protocol support

### ISSUE THREAD

**Issue by ctrlcctrlv** [7sDream/fontfor#3]
The [Kitty graphics protocol](https://sw.kovidgoyal.net/kitty/graphics-protocol.html) is an open standard for displaying graphics in the terminal, similar to [sixels](https://en.wikipedia.org/wiki/Sixel), but not color-palette limited, and comparatively easier to implement, both by TEs and applications.

I'd like to add support for the protocol. Would such support be upstreamed?

**Comment by 7sDream** [comment]
Never hear about this cool protocol, thank you for infomation.

I will happy to see anyone create new useful features based on my codes, and will glad to merge it.

But this feature is a little different, it's somehow out of range of this project……

I'm thinking that could this protocol be add to the [tui](https://github.com/fdehau/tui-rs) crate as a backend or a method, or you can write a new crate for that, it can enpower many other developers, and this project will focus on font render instead of UI things.

In short, I will happy to add kitty protocl support(or accept PRs) after we have a crate for that, instead of write our own kitty impl in this project .

**Comment by ctrlcctrlv** [comment]
Oh sure, I will put the support in its own crate. Working on that now :-)

**Comment by 7sDream** [comment]
After using this tool for years, I believe that the existing Canvas Render functionality largely fulfills the needs of glyph preview within the terminal. We also offer the capability to generate web pages for browser-based previews, if GUI and "picture" is prefered. Considering these, it appears unnecessary for this tiny tool to support terminal graphics protocols.

So this feature will not on my immediate roadmap, hence I'm closing the issue for now.

But feel free to continue the discussion If anyone has further thoughts.
